src/components/common/quantity/Quantity.tsx

```diff
@@ -1,37 +1,116 @@
-import React from "react";
+import React, { useEffect, useState, useCallback } from "react";
 import {
   AutoComplete,
   TextField,
 } from "@madie/madie-design-system/dist/react/";
 import "twin.macro";
 import "styled-components/macro";
+import { CQL } from "cqm-models";
+import * as ucum from "@lhncbc/ucum-lhc";
 
 export interface QuantityProps {
-  quantityValue: number;
-  quantityUnit: any;
-  handleQuantityValueChange: Function;
-  handleQuantityUnitChange: Function;
-  options?: any;
+  quantity: CQL.Quantity;
+  handleQuantityChange?: Function;
```

Comment:
can we rename this to `onChange` or if we really want it to be that specific then `onQuantityChange`? The function that handles the change in the parent would be `handleChange` but best practice for prop names would be some style of `onChange`

Reply:
OK